Modules/BenchPress.Azure/Public/Confirm-RoleAssignment.ps1

```diff
@@ -52,9 +52,8 @@ function Confirm-RoleAssignment {
   }
   Process {
     $params = @{
-      ServicePrincipalName = $ServicePrincipalId
-      RoleDefinitionName   = $RoleDefinitionName
-      Scope                = $Scope
+      ObjectId           = $ServicePrincipalId
```

Comment:
Can you update the `.PARAMETER ServicePrincipalId` to reflect this change ?

Reply:
Updated!